src/enrichment_facility.h

```diff
@@ -72,70 +79,90 @@ class NatUConverter : public cyclus::Converter<cyclus::Material> {
   double feed_, tails_;
 };
 
-///  @class EnrichmentFacility
-///
-///  @section introduction Introduction
-///  The EnrichmentFacility is a simple Agent to agent the enriching of natural
-///  Uranium in a Cyclus simulation. It requests its input recipe (nominally
-///  natural Uranium), and produces any amount of enriched Uranium, given the its
-///  natural uranium inventory constraint and its SWU capacity constraint.
-///
-///  @section requests Requests
-///  The EnrichmentFacility will request from the cyclus::ResourceExchange a
-///  cyclus::Material whose quantity is its remaining inventory capacity and whose
-///  composition is that of its input recipe.
-///
-///  @section acctrade Accepting Trades
-///  The EnrichmentFacility adds any accepted trades to its inventory.
-///
-///  @section bids Bids
-///  The EnrichmentFacility will bid on any request for its output commodity. It
-///  will bid either the request quantity, or the quanity associated with either
-///  its SWU constraint or natural uranium constraint, whichever is lower.
-///
-///  @section extrades Executing Trades
-///  The EnrichmentFacility will execute trades for its output commodity in the
-///  following manner:
-///    #. Determine the trade's quantity and product assay
-///    #. Determine the natural Uranium and SWU requires to create that product
-///    #. Remove the required quantity of natural Uranium from its inventory
-///    #. Extract the appropriate composition of enriched Uranium
-///    #. Send the enriched Uranium as the trade resource
-///
-///  @section gotchas Gotchas
-///  #. In its current form, the EnrichmentFacility can only accept
-///  cyclus::Material having the composition of its input recipe. If a
-///  cyclus::Material of a different composition is sent to it, an exception will
-///  be thrown.
-///
-///  #. During the trading phase, an exception will be thrown if either the
-///  EnrichmentFacility's SWU or inventory constraint is breached.
-///
-///  @section improvements Improvments
-///  The primary improvement to the EnrichmentFacility would be to relax the
-///  requirement that all input material have the in_recipe composition (i.e.,
-///  allow different base enrichments of Uranium).
-///
-///  How would I go about doing so? I'd likely develop an EnrichmentBuffer-type
-///  class that can be queried as to its SWU and natural Uranium capacity.
+///  EnrichmentFacility is a simple Agent that enriches natural
+///  uranium in a Cyclus simulation. It does not explicitly compute
+///  the physical enrichment process, rather it calculates the SWU
+///  required to convert an source uranium recipe (ie. natural uranium)
+///  into a requested enriched recipe (ie. 4% enriched uranium), given
+///  the natural uranium inventory constraint and its SWU capacity
+///  constraint.
+
```

Comment:
need triple slash on this line to maintain continuity of doc comment for doxygen.